### ISSUE TITLE
All resources partially completed, even when they have both alt and long.

### ISSUE THREAD

**Issue by alavatelliMCA** [coyote-team/coyote#417]
Now that we have added the scavenger hunt meta, Coyote thinks that all of our completed images (those that have both alt and long) are partially completed.

**Comment by sinabahram** [comment]
is the ask here to not include scavenger hunt in “completed” status calculation? if so, Staci can handle this one, I feel.

**Comment by stacietaylorcima** [comment]
Hi @alavatelliMCA  I want to get a better understanding of what this feature is supposed to do so that I can figure out how to code it. What does it mean for a resource to be partially completed? What does it mean for a resource to be approved? How do you use this feature in your work flow? Is a resource supposed to be approved when it just has at least 1 alt and 1 long description?

Right now, resources are represented as partially complete when they're not complete. To be complete means that the number representations for a resource is greater than or equal to the number of meta that organization has. 
```
def approved?
  return @approved if defined? @approved
  @approved = complete? && representations.all?(&:approved?)
end

def complete?
    return @complete if defined? @complete
    @complete = representations.count >= organization.meta.count
  end

  def partially_complete?
    return @partially_complete if defined? @partially_complete
    @partially_complete = represented? && !complete?
  end
```

So, organizations have 7 meta right now. If, for example, a resource has 6 long descriptions, it's partially complete. If a resource has 7 long descriptions, it has no status tag. If a resource has 7 long descriptions that are all approved, the resource is approved. So, even if the only meta were alt and long, I'm thinking that it's probably not your intention to have a resource be "approved" if there are 2 approved long descriptions. 

Anyway, let me know what it's intended purpose is and I'll look into it! 

cc @sinabahram

**Comment by stacietaylorcima** [comment]
@alavatelliMCA Hi! I just wanted to follow up on my comments above. :-)

**Comment by sinabahram** [comment]
@flipsasser  we resolved why this was on a recent data model call. Assigning to you for fix.

**Comment by flipsasser** [comment]
This was fixed along with #486

**Comment by flipsasser** [comment]
This was fixed but I forgot to close it
